Makefile

```diff
@@ -15,7 +18,7 @@ install-python: add-ppa
 install-smartcard:
 	sudo apt install -y libusb-1.0-0-dev libpcsclite-dev pcscd pcsc-tools swig
 
-deps: node install-python install-smartcard
+deps: node rust install-python install-smartcard
```

Comment:
since we were going to try to not use make deps at all I don't think this conflicts with the build-system approach so I'm happy to approve this for now to unblock you @benadida